src/main/webapp/app/tasks/task-list/task-list.component.html

```diff
@@ -1,9 +1,48 @@
 <mat-list data-cy="task-list">
   <mat-list-item *ngFor="let task of tasks" class="mat-elevation-z1">
     <mat-icon mat-list-icon>assignment</mat-icon>
-    <h4 mat-line>{{task.name}}</h4>
+    <h4 mat-line>{{ task.name }}</h4>
     <button mat-icon-button aria-label="Delete task" color="primary">
-      <mat-icon aria-label="Delete task" (click)="delete(task)">delete</mat-icon>
+      <mat-icon aria-label="Delete task" (click)="delete(task)"
+        >delete</mat-icon
+      >
+    </button>
+    <label>
+      <input
+        aria-label="mark task as done"
+        type="checkbox"
+        [color]="'primary'"
```

Comment:
using [] in angular means it is an @Input to the component, however this is not true here so `color="primary"` because then the input also does not need to be string-typed

Reply:
Okay that's a good point. Because this was something, I haven't completely understood.